### ISSUE TITLE
Could not find a "psql" to execute

### ISSUE THREAD

**Issue by skolesnyk** [wal-e/wal-e#240]
I've installed daemontools on Centos 7 x64. Is this a problem with path ?

[/]# envdir /etc/wal-e.d/env wal-e backup-push %p
wal_e.main   INFO     MSG: starting WAL-E
        DETAIL: The subcommand is "backup-push".
        STRUCTURED: time=2016-04-20T15:28:54.572057-00 pid=22865
could not find a "psql" to execute
psql: FATAL:  role "root" does not exist
wal_e.main   ERROR    MSG: could not run one or more external programs WAL-E depends upon
        DETAIL: Could not get a connection to the database: note that superuser access
        is required
        Could not run the following programs, are they installed? psql, pv
        STRUCTURED: time=2016-04-20T15:28:54.763567-00 pid=22865

**Comment by amacvar** [comment]
Hi, I'm getting the same message, so i'm assuming start and stop backup commands will not work..

`
> wal-e version
0.9.1
 PostgreSQL 9.4.7 on x86_64-unknown-linux-gnu, compiled by gcc (GCC) 4.8.5 20150623 (Red Hat 4.8.5-4), 64-bit
CentOS Linux release 7.1.1503 (Core) 

> wal-e --aws-instance-profile --s3-prefix=s3://backup backup-push /db/postgres/5432/data
wal_e.main   INFO     MSG: starting WAL-E
        DETAIL: The subcommand is "backup-push".
        STRUCTURED: time=2016-04-20T19:48:56.868954-00 pid=20882
could not find a "psql" to execute
could not find a "psql" to execute
could not find a "psql" to execute
wal_e.operator.backup INFO     MSG: start upload postgres version metadata
        DETAIL: Uploading to s3://backup/basebackups_005/base_000000020000000000000019_00000040/extended_version.txt.
        STRUCTURED: time=2016-04-20T19:48:58.744628-00 pid=20882
wal_e.operator.backup INFO     MSG: postgres version metadata upload complete
        STRUCTURED: time=2016-04-20T19:48:59.723766-00 pid=20882
wal_e.worker.upload INFO     MSG: beginning volume compression
        DETAIL: Building volume 0.
        STRUCTURED: time=2016-04-20T19:48:59.803028-00 pid=20882
wal_e.worker.upload INFO     MSG: begin uploading a base backup volume
        DETAIL: Uploading to "s3://backup/basebackups_005/base_000000020000000000000019_00000040/tar_partitions/part_00000000.tar.lzo".
        STRUCTURED: time=2016-04-20T19:49:00.332021-00 pid=20882
wal_e.worker.upload INFO     MSG: finish uploading a base backup volume
        DETAIL: Uploading to "s3://backup/basebackups_005/base_000000020000000000000019_00000040/tar_partitions/part_00000000.tar.lzo" complete at 6978.42KiB/s.
        STRUCTURED: time=2016-04-20T19:49:01.995380-00 pid=20882
could not find a "psql" to execute
NOTICE:  pg_stop_backup complete, all required WAL segments have been archived`

**Comment by mohitdubey23** [comment]
I am getting the same issue. I have installed wall-e on my ec2 instance to backup my db files to S3. I am not able to understand what is the issue and how to resolve it.

Any help will be appreciated.
Below is the error I am getting

 envdir /etc/wal-e.d/env wal-e backup-push /home/
wal_e.main   INFO     MSG: starting WAL-E
        DETAIL: The subcommand is "backup-push".
        STRUCTURED: time=2016-04-21T15:42:19.372926-00 pid=11198
wal_e.main   ERROR    MSG: could not run one or more external programs WAL-E depends upon
        DETAIL: Could not run the following programs, are they installed? psql
        STRUCTURED: time=2016-04-21T15:42:19.401187-00 pid=11198

**Comment by earsdown** [comment]
I just reverting to wal-e 0.9.0, which made the errors go away.

```
pip install --upgrade 'git+https://github.com/wal-e/wal-e@583ef4a4a89b7ffe204cb3722c25a658dfc19898'
```

**Comment by earsdown** [comment]
I think this issue relates to https://github.com/wal-e/wal-e/issues/234  - recent commit broke wal-e for CentOS 7 users.

**Comment by fdr** [comment]
I went ahead and applied this. Can I get have a tester?

**Comment by amacvar** [comment]
sure - I can help test. What would you like me to do?
(I assumed it was in 0.9.1 already)

**Comment by fdr** [comment]
Sure, download and install the `master` branch and see if it fixes your problem.

**Comment by fdr** [comment]
I think these problems have been resolved in #234. Sorry about that bug, everyone.
